notebooks/official/pipelines/google_cloud_pipeline_components_dataproc_tabular.ipynb

```diff
@@ -0,0 +1,2444 @@
+{
```

Comment:
<p>Explain the component and params.</p><p> </p><p>Params should be on single line. Ex.</p><p> </p><p>def func(param1: str,</p><p>               paramN: str</p><p>):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>